frontend/taipy-gui/src/components/Taipy/DateRange.tsx

```diff
@@ -117,6 +119,11 @@ const DateRange = (props: DateRangeProps) => {
 
     const handleChangeStart = useCallback((v: Date | null) => handleChange(v, true), [handleChange]);
     const handleChangeEnd = useCallback((v: Date | null) => handleChange(v, false), [handleChange]);
+    const analogicRenderers = {
```

Comment:
outside the component ? ie before line 70

Reply:
Done